frontend/src/components/common/header/components/search-bar/search-bar.tsx

```diff
@@ -17,8 +17,7 @@ const SearchBar: FC<Props> = ({ onSearch }) => {
     defaultValues: DEFAULT_SEARCH_PAYLOAD,
   });
 
-  const { field } = useFormControl({ name: 'search', control });
-  const { value } = field;
+  const value = useFormWatch({ name: 'search', control });
```

Comment:
```suggestion
  const value = useForm({ name: 'search', control });
```
should be a useForm with a `onChange` mode